test/integration/default/controls/config.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-control 'TEMPLATE configuration' do
-  title 'should match desired lines'
+control 'TEMPLATE.config.file' do
+  title 'verify the template-formula.conf lines'
```

Comment:
```suggestion
  title 'Verify the configuration file'
```

Reply:
I disagree with the capital `V`, the actual output is:
```
TEMPLATE.config.file: verify the template-formula.conf lines
```

You propose:
```
TEMPLATE.config.file: Verify the configuration file
```

I agree to replace `template-formula.conf lines` with `configuration file` since the file name is given for each check.

Reply:
Yes, that is my proposal. Obviously this is debatable, but the portion after the colon is a statement in its own right, and to me, reads better with an initial capital.

Reply:
We agreed in 2020-01-09 saltstack-formulas meeting to use the capital letter after colon.

Reply:
Hi @baby-gnu, can you make the change to `configuration file` please, to finish this off?

Reply:
ho, something weird during the rebase… I thought I did it.

Thanks.